schema_utils.py

```diff
@@ -229,47 +259,118 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to evaluate.
+            min_value: int. Minimum value of elements.
```

Comment:
No, this is the "Required minimum number of elements."

Reply:
Done